pkgs/desktops/lomiri/dbus-test-runner/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub
+, autoreconfHook, gnome3, intltool, pkg-config, bustle
+, glib, dbus-glib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dbus-test-runner-unstable";
+  version = "2019-01-09";
+
+  src = fetchFromGitHub {
+    owner = "ubports";
+    repo = "dbus-test-runner";
+    rev = "86d63d119566974bd841cacd3202599b3b1a845d";
+    sha256 = "08696ab8ldi22s9ch030sdyzp3m8llszcvhm6span41h782708lm";
+  };
+
+  # TODO better fix?
+  postPatch = ''
+    for Werrorfile in {src,libdbustest,tests}/Makefile.am; do
+      substituteInPlace $Werrorfile \
+        --replace "-Werror" ""
```

Comment:
``cmakeFlags = ''-Wno-error'';``?

Reply:
That package doesn't use cmake so that won't work.

Reply:
Solved, `buildFlags = [ "CFLAGS=-Wno-error" ];` works fine.